### DISCUSSION TITLE
How wouter be compatible with SSR applications

### DISCUSSION THREAD

**Discussion by Trav1sRen** [molefrog/wouter#442] [Q&A]
Is there any equivalent feature in wouter which resembles the `<StaticRouter>` in 'react-router-dom' to be compatible with server-side rendering?
Is this solution still relevant in v3 wouter?
https://gist.github.com/molefrog/70ad57db1acf689b3f4eaf77b7065233

**Comment by molefrog** [comment]
Yes, we support SSR, you can read about it here https://github.com/molefrog/wouter#server-side-rendering-support-ssr
